i18n/active.de.json

```diff
@@ -0,0 +1,123 @@
+{
```

Comment:
I've been thinking about this a bit more and propose to use either `toml` or `yaml` instead of `json` because of two reasons:
- I want comments in my files
- In `json` you have to escape `"` and this sucks.

I'm a bit leaning towards using `toml` because it's the default for `go-1i8n`, but really I'm 0/5.

@kaakaa @jasonblais @scottleedavis Thoughts?

Reply:
My thought: We'd need to make a decision on whether these go to the Translations server or not. If not, I'm 0/5 on the chosen formatting.

Reply:
0/5

Reply:
I'm also on 0/5 too.

> I want comments in my files

What kind of comments do you want to add? My decision point would be here.

> In json you have to escape " and this sucks.

I strongly agree with you. But `yaml` and `toml` seem to have same problem...

Reply:
> I strongly agree with you. But yaml and toml seem to have same problem...

It think `yaml` doesn't uses " for marking strings. See https://gist.github.com/oconnor663/9aeb4ed56394cb013a20

Reply:
Head up: I'm thinking about this and will post a proposal, when I'm done thinking :wink:

Reply:
See #139 for the follow up ticket